statsmodels/robust/scale.py

```diff
@@ -79,11 +75,7 @@ def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0, center=np.median):
     if a.size == 0:
         return np.nan
```

Comment:
a must have at least 1 dim since axis=0.  So we should add a check and raise ValueError if a.ndim == 0